camp_fin/management/commands/import_transactions.py

```diff
@@ -218,9 +218,15 @@ def _get_filing(self, record):
         try:
             pac = models.PAC.objects.get(entity__user_id=state_id)
         except models.PAC.DoesNotExist:
-            pac = models.PAC.objects.get(name=record["Committee Name"])
-            pac.entity.user_id = state_id
-            pac.entity.save()
+            try:
+                pac = models.PAC.objects.get(name=record["Committee Name"])
+            except models.PAC.DoesNotExist:
+                msg = f"PAC with name {record['Committee Name']} does not exist."
+                self.stderr.write(msg)
```

Comment:
why do we want to both write to stderr and raise valueerror?

Reply:
We handle the ValueError in the import and I wasn't sure we'd get the message otherwise?

Reply:
got it. thanks!